Makefile

```diff
@@ -26,8 +26,24 @@ GO111MODULE=on
 all: test manager addonctl
 
 # Run tests
-test: generate fmt vet manifests
```

Comment:
Does this change the report, I want to run all tests as one report.

Reply:
No, it does not. The report is the same.

Reply:
Please post a sample of `make test` output